extension-iac/src/iac_android.cpp

```diff
@@ -211,10 +128,10 @@ extern "C" {
 JNIEXPORT void JNICALL Java_com_defold_iac_IACJNI_onInvocation(JNIEnv* env, jobject, jstring jpayload, jstring jorigin)
 {
     IACCommand cmd;
-    cmd.m_Type = CMD_INVOKE;
-    cmd.m_Payload = StrDup(env, jpayload);
-    cmd.m_Origin = StrDup(env, jorigin);
-    QueueCommand(&cmd);
+    cmd.m_Command = IAC_INVOKE;
+    cmd.m_Payload = (void*)StrDup(env, jpayload);
+    cmd.m_Origin = (void*)StrDup(env, jorigin);
+    IAC_Queue_Push(&g_IAC.m_CmdQueue, &cmd);
```

Comment:
Currently not setting the m_Callback member!

Reply:
The m_Callback wasn't used